pack-plugin.sh

```diff
@@ -1,10 +1,20 @@
 #!/bin/bash
 NAME="yoti-for-drupal-7.x-1.x-edge.zip"
+
+SDK_TAG=$1
+DEFAULT_SDK_TAG='1.2.1'
 SDK_RELATIVE_PATH="sdk"
-curl https://github.com/getyoti/yoti-php-sdk/archive/master.zip -O -L
-unzip master.zip -d sdk
-mv sdk/yoti-php-sdk-master/src/* sdk
-rm -rf sdk/yoti-php-sdk-master
+
+if [ "$SDK_TAG" = "" ]; then
+    SDK_TAG=$DEFAULT_SDK_TAG
```

Comment:
Do you think it would be useful to add something like `echo "no tag provided, using default tag of $SDK_TAG"`?

Reply:
If you think it will be useful, as I am already displaying the tag used after that statement

`echo "Pulling PHP SDK TAG $SDK_TAG.zip ..."`

Reply:
Yer I guess it's not essential information.

I was trying to think of when someone might want to know when the default version has been used or not, but can't really think of when that would be needed